### PR TITLE
Split e2e tests

### PR DESCRIPTION
**Pull request by yanivefraim**
This is a POC for splitting e2e tests into two: tests starting with the letters: `a-H` and `I-Z`.
It should wait for this [PR](https://github.com/wix/yoshi/pull/1235) to be merged first (and a new Yoshi version to be released)

### CODE REVIEW

File: protractor1.conf.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports.config = {
+  specs: [
+    'test/{A..H}**/**/*.e2e.js',
+    'src/{A..H}**/**/*.e2e.js',
+    'test/{a..z}**/**/*.e2e.js',
```
Comment (gilmeir) [inline protractor1.conf.js]: Here are the combinations as I understand we want to cover. 
Here you catch all a-z

protractor1.conf.js:
1. a-h
2. A-H

protractor2.conf.js:
3. i-z
4. I-Z 

Also, I suggest renaming the files protractor-a-to-i.conf.js, protractor-h-to-z.conf.js
so it's a bit more meaningful

**Comment by mosheKerbel** [comment]
To make sure all tests are executed i suggest after the next normal build, currently it's building one, we count down the tests executed in applitools and compare it to another PR without the split.

**Comment by mosheKerbel** [comment]
@yanivefraim notice that i pushed force to resolve the conflict

**Comment by mosheKerbel** [comment]
ok applitools executed all tests - 528, as it should.
i'm just checking now about the failing tests - in test1
i suspect the latest Yoshi brought us other updated react dependencies that casuse there warnings which fail the tests

**Comment by mosheKerbel** [comment]
> ok applitools executed all tests - 528, as it should.
> i'm just checking now about the failing tests - in test1
> i suspect the latest Yoshi brought us other updated react dependencies that casuse there warnings which fail the tests

fixed in other PRs and test1 now pass
merging.

**Pull request merged**